docs/guides/controller-autoconfiguration.md

```diff
@@ -36,34 +37,53 @@ This allows:
 
 Having automatically configured joypads makes it a lot easier to navigate the RetroArch Menu with the joypad. This is very convenient when running RetroArch on a game console, where a keyboard and a mouse are not always available. It is also what makes RetroArch suitable to build your own game console using Lakka or a similar OS.
 
+## Before you begin
+
+Make sure that you run the latest version of RetroArch, to generate a file name and file content via `Settings` -> `Input` -> `RetroPad Binds` -> `Port 1 Controls` -> `Save Controller Profile` that is up to date with our current policies.
+
+For example, RetroArch 1.16.0 incorrectly added these lines by default:
+```
+input_gun_trigger_mbtn = "1"
+input_gun_offscreen_shot_mbtn = "2"
+```
+
+
 ## Installing or updating joypad profiles
 
 ![downloading joypad profiles](../image/retroarch/input/update-joypads.jpg)
 
-The set of joypad profiles used by RetroArch can be downloaded and updated from the menu. Go to `Main Menu` -> `Online Updater` -> `Update Autoconfig Profiles` to get the latest version of the profile pack.
+The set of joypad profiles used by RetroArch can be downloaded and updated from the menu. Go to `Main Menu` -> `Online Updater` -> `Update Controller Profiles` to get the latest version of the profile pack.
 
 A message will appear at the bottom of the screen showing the download progress and the extraction of the archive.
 
+
 ## Generating a joypad profile
 
 If your joypad is not recognized by RetroArch even after updating the profiles, you can generate a profile from the menu.
 
-
 1. Unplug all the other joypads
-2. Use `Settings` -> `Input` -> `User 1 Binds` -> `User 1 Bind All`. Make sure that your mapping is perfect by testing every button in the menu and in some games.
-3. Use `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Save Autoconfig`
-4. The new profile will be saved to your disk.
-5. Use `Settings` -> `Inputs` -> `User 1 Binds` -> `User 1 Bind Default All` to clear manual bindings and rely on the new profile
-6. Unplug your joypad an re-plug it. See if it is auto configured.
+2. For Android, run the [Input descriptors](#android) step first.
```

Comment:
Does this have to be specific to Android? I feel like defining the Input Descriptors could be useful across all platforms.

Reply:
You are right. I added the wrong text before to the link. But i correctet it now in https://github.com/libretro/docs/pull/910/commits/5d8f06e7bc41416c11d6ea14b541ce022358fc15